### PR TITLE
Functions to send basic image and video events

### PR DESCRIPTION
**Pull request by rxl881**

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -450,6 +450,28 @@ func (cli *Client) SendText(roomID, text string) (*RespSendEvent, error) {
...
+		ImageMessage{
+			MsgType: "m.image",
+			Body:    body,
+			URL:     url,
+		})
+}
+
+// SendVideo sends an m.room.message event into the given room with a msgtype of m.video
+// See https://matrix.org/docs/spec/client_server/r0.2.0.html#m-video
+func (cli *Client) SendVideo(roomID, body string, url string) (*RespSendEvent, error) {
```
Comment (Kegsay) [inline client.go]: Function signature is weird, can we change this to just be:
```go
SendVideo(roomID, body, url string)
```

please? Ditto on `:455`.

File: events.go [hunk 2]
```diff
@@ -52,6 +52,25 @@ type ImageInfo struct {
 	Size     uint   `json:"size"`
 }
 
+// VideoInfo contains info about a video
```
Comment (Kegsay) [inline events.go]: Link to docs please.

File: events.go [hunk 3]
```diff
@@ -52,6 +52,25 @@ type ImageInfo struct {
...
+	Mimetype      string    `json:"mimetype"`
+	ThumbnailInfo ImageInfo `json:"thumbnail_info"`
+	ThumbnailURL  string    `json:"thumbnail_url"`
+	Height        uint      `json:"h"`
+	Width         uint      `json:"w"`
+	Duration      uint      `json:"duration"`
+	Size          uint      `json:"size"`
+}
+
+// VideoMessage is an m.video event
```
Comment (Kegsay) [inline events.go]: Link to docs please.

**Comment by Kegsay** [comment]
Otherwise LGTM, go ahead and merge, I don't need to see this again.

**Comment by rxl881** [comment]
@Kegsay - Is this good to go now? Thanks.

**Comment by Kegsay** [comment]
LGTM

**Comment by rxl881** [comment]
Merged directly to master in #26

**Pull request closed without merging**